### DISCUSSION TITLE
Newsletter interne et ajout de fonctionnalités

### DISCUSSION THREAD

**Discussion by PaulMalliet** [OFCE/ofce#10] [Ideas]
Dites, ce serait sans doute pas mal qu'on mette en place une newsletter interne sur les nouvelles fonctionalités introduites dans le package ofce. Pour une prochaine je pense  déjà aux wp en anglais et au theme.gt pour les tables.   
Vous en pensez quoi ?

**Comment by PaulMalliet** [comment]
Proposition pour release 1.1: 

- Ajout d'un format Wp en anglais 
- Ajout d'une fonction pour mettre au style _ofce_ les tableaux générés au format [Gt](https://gt.rstudio.com/) sous R 
- Modification du style pour le format _blog_ via `ofce-quarto-extensions` avec ajout d'un exemple de tableau dans le template accessible via la fonction `setup_blog`

**Comment by xtimbeau** [comment]
ce serait super de mettre en vignette du package le guide de publication
